src/rest-client.ts

```diff
@@ -498,12 +509,14 @@ export class RestClient extends BaseRestClient {
   getMaxLoan(
     instId: string,
     mgnMode: MarginMode,
-    mgnCcy?: string
+    mgnCcy?: string,
+    ccy?: string
```

Comment:
I think we might need to do the params-object method here with a deprecation on the current one. Might also be good to update the type to suggest mgnCcy can be undefined too, in case someone wants to set `ccy` and not `mgnCcy`
```suggestion
    mgnCcy?: string | undefined,
    ccy?: string
```